src/power.jl

```diff
@@ -0,0 +1,12 @@
+"""
+    Power(exponent) <: Transformation
+
+Raise the data by the given `exponent`.
+"""
+struct Power <: Transformation
+    exponent::Real
+end
+
+function _transform!(x::AbstractArray{T}, P::Power; kwargs...) where T <: Real
+    x[:] = x.^ P.exponent
```

Comment:
I get `InexactError` if I do
```
x = [1, 2, 3]
p = Power(1.5)
p(x)
```
because it tries to do `Int()` on the elements of `p(x)`.